docs/data-protection-and-privacy-80e76fd.md

```diff
@@ -9,6 +9,13 @@ For general information about data protection and privacy on SAP Business Techno
 > ### Note:  
 > SAP does not provide legal advice in any form. SAP software supports data protection compliance by providing security features and data protection-relevant functions, such as blocking and deletion of personal data. In many cases, compliance with applicable data protection and privacy laws is not covered by a product feature. Furthermore, this information should not be taken as advice or a recommendation regarding additional features that would be required in specific IT environments. Decisions related to data protection must be made on a case-by-case basis, taking into consideration the given system landscape and the applicable legal requirements. Definitions and other terms used in this documentation are not taken from a specific legal source. Handle personal data with care. You as the data controller are legally responsible when processing personal data.
 
+> ### Note:  
+> SAP Cloud Logging is not designed to support the collection, processing, and storage of personal or business data.  
+We ask you take appropriate measures to ensure that no personal or business data is sent to our service.
```

Comment:
Take appropriate measures to ensure that no personal or business data is sent to the service.